GliaWidgets/SecureConversations/SecureConversations.Coordinator.swift

```diff
@@ -139,7 +149,7 @@ extension SecureConversations {
                     with: conf, cancelled: cancelled
                 )
             case .transcriptRequested:
-                navigateToTranscript()
+                _ = navigateToTranscript()
```

Comment:
You can use `@discardableResult`

Reply:
Fixed [here](https://github.com/salemove/ios-sdk-widgets/pull/507/files#diff-6a8aade246430e27807b109ce4a24745400410668b8ce4eb1c0477eeea0e14c6R239)